components/event-sources/Makefile

```diff
@@ -16,6 +16,8 @@ APP_NAME = event-sources
 APP_PATH = components/event-sources
 BUILDPACK = eu.gcr.io/kyma-project/test-infra/buildpack-golang:go1.12
 SCRIPTS_DIR = ../../common/makefiles
+
+patched_trace = vendor/go.opencensus.io/trace/trace.go
```

Comment:
I feel like this variable belongs closer to where it's actually used, instead of mixed with the variables used by the shared Makefile.

Reply:
Sure, why not.